python/semantic_kernel/agents/agent.py

```diff
@@ -75,7 +130,7 @@ def _configure_plugins(cls, data: Any) -> Any:
         return data
 
     @abstractmethod
-    async def get_response(self, *args, **kwargs) -> ChatMessageContent:
+    async def get_response(self, *args, **kwargs) -> AgentResponseItem[ChatMessageContent]:
```

Comment:
can we also unify the params passed to this, to make this an actual abstraction?

Reply:
Yeah, I think we can consolidate these now.
```suggestion
    async def get_response(self, *, message: str | ChatMessageContent, thread: AgentThread | None = None, **kwargs) -> AgentResponseItem[ChatMessageContent]:
```